CONTRIBUTORS.md

```diff
@@ -0,0 +1,27 @@
+## Contributors
+
+This file lists the contributors to the PureScript documentation project, and the terms under which their code is licensed.
+
+### Contributors using Standard Terms
+
+Contributors listed here agree to license their contributions under the following terms:
+
+> My existing contributions and all future contributions until further notice are Copyright {Name}, and are licensed to the owners and users of the PureScript documentation project under the terms of the {License}.
+
+By adding your name to the list below, you agree to license your contributions under these following terms.
+
+If you would prefer to use different terms, please use the section below instead.
+
+| Username | Name | License |
+| :------- | :--- | :------ |
+| [@LiamGoodacre](https://github.com/LiamGoodacre) | Liam Goodacre | [MIT license](http://opensource.org/licenses/MIT) |
```

Comment:
Choose whichever license you like, of course, but I'm going to choose a creative commons license, probably the one I use for the book, and I'm going to recommend others do the same, since I think it's more appropriate for written work.